packages/giveaway/package.json

```diff
@@ -55,44 +55,40 @@
         }
     },
     "dependencies": {
-        "@openzeppelin/contracts-upgradeable": "^4.9.0"
+        "@openzeppelin/contracts-upgradeable": "^4.9.3"
```

Comment:
does upgrading OZ change the sol files ?

Reply:
Nothing that affects our code, specially becase we don't use ERC2771Context. 
In any case we need to fix the version (without ^) because according to the yarn.lock file we used 4.9.3 before this change.

Reply:
Just because of the small changes, if we don't redeploy (which I think is not necessary right now) it wont fully match